docs/development/release-schedule.md

```diff
@@ -0,0 +1,39 @@
+# Supported Versions
+
+Every ILIAS version will be **fully supported** until the end of the year after
+the year it was released in. **Fully supported** means that we are fixing every
+kind of issue that is reported for the release according to our bugfixing process.
+*E.g.: A usability issue, reported for ILIAS 8 (released 2022), can be reported
+in august 2023 and is eligible for a fix then.*
+
+Every ILIAS version will then gain **limited support** for an additional year
+after that. **Limited support** means that we are fixing security issues, fatal
+errors and exceptions. *E.g.: A security issue, reported for ILIAS 8 (release 2022),
+can be reported august 2024 and is eligible for a fix then. A malfunction that does
+make the program crash, reported at the same moment, won't be eligible for a fix.*
+
+
+## Timeline per Version
+
+With that support schedule, every version will have (roughly) the following timeline:
+
+* Nov, year of release - 2: jour fixe for project planning
+* Apr, year of release - 1: feature freeze
+* Oct, year of release - 1: start of the beta phase
+* Mar, year of release: release of the new version
+* Dec, year of release + 1: end of full support
+* Dec, year of release + 2: end of limited support
+
```

Comment:
```suggestion
```
             20X0                    20X1                    20X2                    20X3

             │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │
             │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │ │
             │                       │                       │                       │
             │                       │                   ┌─┬─┴───────┬───────────┬───┴─────┬─────
ILIAS Z      │                       │                   │P│       FF│         CC│B        │R
             │                       │                   └─┴─┬───────┴───────────┴───┬─────┴─────
             │                       │                       │                       │
             │                   ┌─┬─┴───────┬───────────┬───┴─────┬─────────────────┤
ILIAS Y      │                   │P│       FF│         CC│B        │R               E│
             │                   └─┴─┬───────┴───────────┴───┬─────┴─────────────────┤
             │                       │                       │                       │
         ┌─┬─┴───────┬───────────┬───┴─────┬─────────────────┼─ ── ── ── ── ── ── ── │
ILIAS X  │P│       FF│         CC│B        │R               E│                      D│
         └─┴─┬───────┴───────────┴───┬─────┴─────────────────┼─ ── ── ── ── ── ── ── │
             │                       │                       │                       │


             P:  Jour Fixe for Project Planing
             FF: Feature Freeze
             CC: Coding Completed
             B:  Start of the Beta Phase
             R:  Release of the new Version
             E:  End of Full Support
             D:  End of Limited Support
```
```

Reply:
nah, hat wohl doch nicht so geklappt :D

Reply:
Liegt aber eher an der suggestion glaube ich, bei mir sieht das so aus: 
![image](https://user-images.githubusercontent.com/6661332/145164450-ded73859-7ddb-4954-8c22-c44ed2f6d2ee.png)

Reply:
oder so ```
             20X0                    20X1                    20X2                    20X3

             | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | |
             | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | |
             |                       |                       |                       |
             |                       |                   +-+-+-------+-----------+---+-----+-----
ILIAS Z      |                       |                   |P|       FF|         CC|B        |R
             |                       |                   +-+-+-------+-----------+---+-----+-----
             |                       |                       |                       |
             |                   +-+-+-------+-----------+---+-----+-----------------+
ILIAS Y      |                   |P|       FF|         CC|B        |R               E|
             |                   +-+-+-------+-----------+---+-----+-----------------+
             |                       |                       |                       |
         +-+-+-------+-----------+---+-----+-----------------+- -- -- -- -- -- -- -- |
ILIAS X  |P|       FF|         CC|B        |R               E|                      D|
         +-+-+-------+-----------+---+-----+-----------------+- -- -- -- -- -- -- -- |
             |                       |                       |                       |


             P:  Jour Fixe for Project Planing
             FF: Feature Freeze
             CC: Coding Completed
             B:  Start of the Beta Phase
             R:  Release of the new Version
             E:  End of Full Support
             D:  End of Limited Support
```

Reply:
jetzt tuts :D